parm/config/gfs/config.base

```diff
@@ -178,6 +178,7 @@ export DO_WAVE="NO"
 export DO_OCN="NO"
 export DO_ICE="NO"
 export DO_AERO="NO"
+export DO_PREP_AERO_OBS="YES"
```

Comment:
Is there a need for the job to have its own on/off switch? If so, that's fine; but we also want to avoid having too many single-job switches.

Reply:
The idea was if the obs are already prepared in the DMPDIR, there is no need for this. An offline discussion with @aerorahul along these lines was just had. If there is a desire to move all of these observation prep jobs upstream of the workflow we can do so _eventually_ but without this capability in the workflow soon, GFSv17 development is at risk.